app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -708,8 +710,30 @@ export default {
       }
       this.$nextTick(() => this.$refs.messageInput.focus());
     },
+    onRichContentEditorSelectionChange(selection) {
+      this.richContentEditorSelection = selection;
+    },
+    insertEmoji(emoji, selectionStart, selectionEnd) {
```

Comment:
Instead of doing this here can we

1. Add a prop for `Editor.vue` update-selection-with which will accept text
2. Create a watch method over the `updateSelectionWith`
3. Check the `insertCannedResponse` method created in https://github.com/chatwoot/chatwoot/pull/5880/files to see how to insert text at selection

@pranavrajs Thoughts?

Reply:
Done https://github.com/chatwoot/chatwoot/pull/5865/commits/77a6621e7e1b17dd2b07a2a5bfb578067638289f